### PR TITLE
feat: update technical screening template

### PR DESCRIPTION
**Pull request by valerydluski**
**🟢 Add `deploy` label if you want to deploy this Pull Request to staging environment**

#### 🧑‍⚖️ Pull Request Naming Convention

- Title should follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- Do not put issue id in title
- Do not put WIP in title. Use [Draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/) functionality
- Consider to add `area:*` label(s)

* [x] I followed naming convention rules

---

#### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Test Case
- [x] Documentation update
- [ ] Other

#### 🔗 Related issue link

[issue](https://github.com/rolling-scopes-school/tasks/issues/1419)

#### 💡 Background and solution

image

#### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Database migration is added or not needed
- [x] Documentation is updated/provided or not needed
- [x] Changes are tested locally

### CODE REVIEW

**Review by AlreadyBored — APPROVED** [review]

File: client/src/data/interviews/technical-screening.tsx [hunk 1]
```diff
@@ -289,6 +289,56 @@ const theoryQuestions = [
...
+    title: 'Variables',
+    topic: 'JavaScript Fundamentals',
+  },
+  {
+    id: 'numbers',
+    title: 'Number & Math methods',
+    topic: 'JavaScript Fundamentals',
+  },
+  {
+    id: 'Strings',
```
Comment (aaliakseyenka) [inline client/src/data/interviews/technical-screening.tsx:308]: probably let's use same camelcase for all `ids`

File: client/src/data/interviews/technical-screening.tsx [hunk 2]
```diff
@@ -289,6 +289,56 @@ const theoryQuestions = [
...
+    title: 'String methods',
+    topic: 'JavaScript Fundamentals',
+  },
+  {
+    id: 'stringTemplates',
```
Comment (aaliakseyenka) [inline client/src/data/interviews/technical-screening.tsx:313]: just curious should it be really diffrent topic than just `string`?

File: client/src/data/interviews/technical-screening.tsx [hunk 3]
```diff
@@ -289,6 +289,56 @@ const theoryQuestions = [
...
+    title: 'String templates',
+    topic: 'JavaScript Fundamentals',
+  },
+  {
+    id: 'ternaryOperator',
```
Comment (aaliakseyenka) [inline client/src/data/interviews/technical-screening.tsx:318]: separate question for ternary operator 😄

**Review by aaliakseyenka — APPROVED** [review]

File: client/src/data/interviews/technical-screening.tsx (hunk 2 above)
Comment (valerydluski) [reply client/src/data/interviews/technical-screening.tsx]: fixed

File: client/src/data/interviews/technical-screening.tsx (hunk 1 above)
Comment (valerydluski) [reply client/src/data/interviews/technical-screening.tsx]: done

File: client/src/data/interviews/technical-screening.tsx (hunk 3 above)
Comment (valerydluski) [reply client/src/data/interviews/technical-screening.tsx]: Thanks mate, I updated this question

**Pull request merged**